build.xml

```diff
@@ -271,6 +271,13 @@
             <arg value="index.js"/>
         </exec>
     </target>
+    
+    <target name="generate-images-py">
+        <exec executable="python3" logerror="true" dir="${basedir}/utils/examples" failifexecutionfails="true" failonerror="true">
+            <arg value="generate-examples.py"/>
+            <arg value="--clean"/>
```

Comment:
```suggestion
            <arg value="--clean"/>
            <arg value="--verbose"/>
```

Reply:
Suggestion to run on verbose mode (= log INFO level) to see Verovio version and rendered file names in output